ads1015/src/main/java/com/example/upm/androidthings/driversamples/ADCActivity.java

```diff
@@ -88,14 +93,15 @@ public void run() {
             Executors.newSingleThreadScheduledExecutor().schedule(new Runnable() {
```

Comment:
In android handler.postdelayed() is used to execute a command after some specified time and in java they were using Executers to execute a command after specific time. I just used whatever is there in the java examples.